proposals/exception-handling/Exceptions.md

```diff
@@ -115,8 +115,8 @@ A try-catch block contains zero or more `catch` blocks and zero or one
 `catch_all` block. All `catch` blocks must precede the `catch_all` block, if
 any. The `catch`/`catch_all` instructions (within the try construct) are called
 the _catching_ instructions. There may not be any `catch` or `catch_all` blocks
-after a `try`, in which case the whole `try` block is effectively a regular
-block.
+after a `try`, in which case the `try` block does not catch any exceptions and
+rethrows them.
```

Comment:
```suggestion
after a `try`, in which case the `try` block does not catch any exceptions.
```
(I think the rethrow part is a bit off, since technically, a non-catch isn't the same as a rethrow.)